runtime/lua/vim/lsp.lua

```diff
@@ -542,6 +542,12 @@ function lsp.start_client(config)
       client_ids[client_id] = nil
     end
 
+    if signal ~= 0 then
+      vim.schedule(function()
+        vim.notify(string.format("Client %s crashed with exit code %s", client_id, signal))
```

Comment:
Do you think it would make sense to set the log level to warn or even error ?

Reply:
I did now, but currently the behavior of anything but error is to print to :messages